img_proof/scripts/cli_utils.py

```diff
@@ -73,6 +73,24 @@ def archive_history_item(item, destination, no_color):
         )
 
 
+def cli_process_cpu_options(cxt, param, value):
+    cpu_options = []
+    if value:
+        try:
+            for cpu_option in value.split(','):
+                key, val = cpu_option.split('=', 1)
+                option = {
+                    'key': key,
+                    'value': val
+                }
+                cpu_options.append(option)
+        except Exception as e:
+            raise click.BadParameter(
+                "Issue with cpu-option parameter: %s" % e
```

Comment:
Nitpick: wording here could be more clear. I.e. something like '"--cpu-option" parameter is invalid: ...'. Also, could be helpful to provide a valid example input.

Reply:
Updated with the additional info suggested. Thanks!